modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -408,6 +417,18 @@ export default function ObjectFieldFormBase({
 		}
 	}, [reloadPicklistSingleSelect]);
 
+	useEffect(() => {
+		const makeFetch = async () => {
+			if (values.id) {
+				setObjectFieldRequired(
+					(await API.getObjectField(values.id)).required
+				);
+			}
+		};
+
+		makeFetch();
+	}, [values.id]);
+
```

Comment:
Hey @igor-franca, Hi Igor, as we discussed, this `useEffect` will not be necessary because we have this information in the first request we make when editing a field, so we can take it from there instead of making a new request